backend/app/models/listeners.py

```diff
@@ -0,0 +1,88 @@
+from datetime import datetime
+from pydantic import Field, BaseModel
+from typing import Optional, List, Union
+from app.models.pyobjectid import PyObjectId
+from app.models.mongomodel import MongoModel
+from app.models.users import UserOut
+
+
+class Repository(MongoModel):
+    """Reference to a repository associated with Event Listener/Extractor."""
+
+    repository_type: str = "git"
+    repository_url: str = ""
+
+
+class ExtractorInfo(BaseModel):
+    """Currently for extractor_info JSON from Clowder v1 extractors for use with to /api/extractors endpoint."""
+
+    author: str  # Referring to author of listener script (e.g. name or email), not Clowder user
+    process: dict
+    maturity: str = "Development"
+    contributors: List[str] = []
+    contexts: List[dict] = []
+    repository: List[Repository] = []
+    external_services: List[str] = []
+    libraries: List[str] = []
+    bibtex: List[str] = []
+    default_labels: List[str] = []
+    categories: List[str] = []
+    parameters: List[dict] = []
+
+
+class EventListenerBase(BaseModel):
```

Comment:
Is this supposed to inherit from `ExtractorInfo`?

Reply:
Sounds good. We can try to keep both. Maybe add a few more comments explaining the difference? As long as we can submit/trigger with current extractors, we are good short term. I don't think this gets in the way of that?